ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/wholeBodyHardwareControl/AvatarLowLevelOutputProcessor.java

```diff
@@ -164,7 +164,7 @@ public JointDesiredOutputListReadOnly getProcessedDesiredOutput()
       return processedDesireds;
    }
 
-   public DoubleProvider getMasterGain()
+   public YoDouble getMasterGain()
```

Comment:
@stefanfasano This is the main change, allowing other things to set masterGain. i.e. the operator.